src/jest.ml

```diff
@@ -129,6 +129,7 @@ end = struct
 end
 
 module Runner (A : Asserter) = struct
+  type timeout = int
```

Comment:
Not very important, but why use a type alias here instead of just `int` directly?

Reply:
Fixed.

I misunderstand your previous comment.